app/lib/methods/audioPlayer.ts

```diff
@@ -42,6 +48,32 @@ class AudioPlayer {
 				try {
 					await this.audioQueue[audioKey]?.stopAsync();
 					this.audioPlaying = '';
+
+					const [msgId, rid] = audioKey.split('-');
+
+					const msg = await getMessageById(msgId);
+
+					if (msg) {
+						const db = database.active;
+						const [message] = await db
+							.get('messages')
+							.query(
+								Q.where('rid', rid),
+								Q.experimentalSortBy('ts', Q.asc),
+								Q.where('ts', Q.gt(moment(msg.ts).valueOf())),
+								Q.experimentalTake(1)
+							)
+							.fetch();
+
+						if (message && message.attachments) {
+							const file = message.attachments[0];
+							const nextAudioInSeqKey = `${message.id}-${rid}-${getFilePathAudio(file)}`;
```

Comment:
As now, you need to reuse the same format to generate a key, pls, create a function outside the class receiving those parameters.

```js
const getAudioKey = ({ msgId, rid, uri }: { msgId: string; rid: string; uri: string }) => `${msgId}-${rid}-${uri}`;
```

```suggestion
							const nextAudioInSeqKey = getAudioKey({ msgId: message.id, rid, uri: getFilePathAudio(file) });
```